src/meson.build

```diff
@@ -31,7 +31,14 @@ if get_option('enable_d3d9')
   subdir('d3d9')
 endif
 
+if get_option('enable_d3d8')
+  if not get_option('enable_d3d9')
+    error('D3D9 is required for D3D8.')
+  endif
+  subdir('d3d8')
```

Comment:
should we even bother building 64-bit d3d8 at all?

Reply:
Maybe not by default but it's useful for replaying apitraces.